tests/testapps/CTCatalog/CTCatalog/Tests/CTCParagraphStyleTestViewController.mm

```diff
@@ -252,12 +252,12 @@ - (void)drawTests {
 
     // Add base writing direction based on selection
     CTWritingDirection direction;
-    if ([_baseWritingDirection isEqualToString:@"kCTWritingDirectionNatural"]) {
-        direction = kCTWritingDirectionNatural;
+    if ([_baseWritingDirection isEqualToString:@"kCTWritingDirectionRightToLeft"]) {
+        direction = kCTWritingDirectionRightToLeft;
```

Comment:
Yes, forgot to comment. This was a small change I found while testing the code. AJ has fixed this on his latest CR so this should go away once I rebase.

---
In reply to: [80768900](https://github.com/Microsoft/WinObjC/pull/1040#discussion_r80768900) [](ancestors = 80768900)